act-framework.bs

```diff
@@ -42,10 +42,8 @@ Rule Description {#structure-description}
 
 Each ACT Rule MUST have a description that:
 
-* Is in plain language;
-* Explains the accessibility requirement being tested;
-* Lists any limitations and assumptions for the test;
-* Explains the failure conditions and their reasons.
+* Is in plain language and provides a brief description of what the rule does.
```

Comment:
"a rule must have a description that provides a brief description of what the rule does" sounds a bit tautological 😉.
I'd suggest "provides a summary" or "provides a short explanation" instead.